### PR TITLE
Entrack

### PR DESCRIPTION
**Pull request by ppoulin91**

### CODE REVIEW

File: article/_posts/2019-11-21-entrack.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- Predicts the parameters for a unimodal distribution on the sphere (Fisher von Mises)
+- Uncertainty quantification
+- Maximum-entropy regularization as loss function instead of NLL to enforce non-zero uncertainty and robustness to noise
+
+# Method
+
+**Model**: Simple Multi-Layer perceptron
+
+**Input**: 
+- Block of 3x3x3 voxels of DWI data fitted with sphericla harmonics of order 4 (15 values per voxel = 405 features)
```
Comment (AntoineTheb) [inline article/_posts/2019-11-21-entrack.md]: ~sphericla~ spherical

File: article/_posts/2019-11-21-entrack.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+> ![](/article/images/entrack/fisher_dist.jpg)
+
+> (Taken from [Wikipedia](https://en.wikipedia.org/wiki/Von_Mises%E2%80%93Fisher_distribution))
+
+> ![](/article/images/entrack/von_mises_sphere.jpg)
+
+> (Taken from [[1]])
+
+The Fisher von Mises distrbution is analogous to a Gaussian on the Sphere. It has a mean parameter $$\mu$$ and a *concentration* parameter $$\kappa$$.
```
Comment (AntoineTheb) [inline article/_posts/2019-11-21-entrack.md]: ~distrbution~ distribution

File: article/_posts/2019-11-21-entrack.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+![](/article/images/entrack/equation4.jpg)
+
+![](/article/images/entrack/equation3.jpg)
+
+The model is optimized using SGD and gradient clipping.
+
+# Experiments
+
+- Training: 1 HCP subject, tracked using a probablistic algorithm.
```
Comment (AntoineTheb) [inline article/_posts/2019-11-21-entrack.md]: ~probablistic~ probabilistic

File: article/_posts/2019-11-21-entrack.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    - Output metrics: Valid Bundles (VB), Invalid Bundles (IB), Valid Connections (VC), Overlap (OL), Overreach (OR), F1-score (using overlap + overreach)
+
+## Results
+
+![](/article/images/entrack/table1.jpg)
+
+> The Entrack model achieves scores comparable to the other works, indicating its usefulness as predictive model for fiber tracking, in addition to its benefits of uncertainty quantification, and outlier detection.
+
+
+- The model is also trained on the TM data and used to color the training streamlines according to the total probability ((a) and (b) or local uncertainty ((c) and (d)).
```
Comment (jhlegarreta) [inline article/_posts/2019-11-21-entrack.md]: TM (tractometer) has not been defined

Comment (jhlegarreta) [reply article/_posts/2019-11-21-entrack.md]: Addressed in f7d3ad4.

**Pull request merged**